CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [#1430](https://github.com/bbatsov/rubocop/issues/1430): Add `--except` option for disabling cops on the command line. ([@jonas054][])
 * [#1506](https://github.com/bbatsov/rubocop/pull/1506): Add auto-correct from `EvenOdd` cop. ([@blainesch][])
 * [#1507](https://github.com/bbatsov/rubocop/issues/1507): `Debugger` cop now checks for the Capybara debug methods `save_and_open_page` and `save_and_open_screenshot`. ([@rrosenblum][])
+* [#1271](https://github.com/bbatsov/rubocop/issues/1272): `Lambda` cop does auto-correction. ([@lumeet][])
```

Comment:
Typo in the URL, should be `1271`. And you need to add a link for yourself at the bottom of the file.